### ISSUE TITLE
CUDA/GPU architecture requirements

### ISSUE THREAD

**Issue by mjstrumillo** [SuperBianC/scMulan#1]
hi, can you provide minimum system/GPU/CUDA requirements for installation? My understanding is the model is pre-trained so if no finetuning is required, can the tutorials be run without the GPU support? Thanks!

**Comment by SuperBianC** [comment]
Yes, it can be run with CPU only, I added settings for using CPU in this tutorial [https://github.com/SuperBianC/scMulan/blob/main/Tutorial-cell_type_annotation.ipynb](url). But only using CPU would be very slow. Using GPUs would be at least 10x faster.
